<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRuleForm</class>
 <widget class="QDialog" name="AddRuleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Traffic type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTrafficType">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>input</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>output</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Src addr</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSrcAddr">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Src port</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSrcPort">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDstAddr">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDstPort">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dst port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dst addr</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Protocol</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxProtocol">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Any</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>UDP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ICMP</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAction">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Deny</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Accept</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSrcAdrAny">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Any</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDstAAny">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Any</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSrcPortAny">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Any</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDstPortAny">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Any</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
